vocabularies/identifier-type.ttl

```diff
@@ -0,0 +1,122 @@
+ dct:modified "2020-03-24T14:57:04+10:00"^^xsd:dateTime ;
+ dct:source "Compiled by the Geological Survey of Queensland." ;
+ skos:definition "Types of Identification documents and numbers used in Queensland as acceptable proof of identity for individuals, businesses, and companies."@en ;
+ skos:prefLabel "Identifier Type"@en .
+
+<http://linked.data.gov.au/org/gsq>
+ a sdo:Organization ;
+ sdo:name "Geological Survey of Queensland" ; 
+ .
+
+idt:abn a skos:Concept ;
+ skos:altLabel "Australian business number"@en ;
+ skos:definition "The Australian Business Number (ABN) is a unique number used to identify business names and companies. The Australian Business Register (ABR) provides ABNs. Source:ASIC"@en ;
+ skos:inScheme idt:conceptScheme ;
+ skos:prefLabel "ABN"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:acn a skos:Concept ;
+ skos:altLabel "Australian company number"@en ;
+ skos:definition "Every company in Australia is issued with a unique, nine-digit number when registered. This is an Australian Company Number (ACN) and must be displayed on all company documents. Source:ASIC"@en ;
+ skos:inScheme idt:conceptScheme ;
+ skos:prefLabel "ACN"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:australian-visa a skos:Concept ;
+ skos:definition "A conditional permit granted to a foreign national by the Australian Government conveying rights of entry, residence, or work within the country."@en ;
+ skos:inScheme idt:conceptScheme ;
+ skos:prefLabel "Australian visa"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:birth-certificate a skos:Concept ;
+ skos:definition "A vital record that documents the birth of a child. Birth certificates may be the original document certifying the circumstances of the birth or to a certified copy or representation of the ensuing registration of that birth. "@en ;
+ skos:inScheme idt:conceptScheme ;
+ skos:prefLabel "Birth certificate"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:certificate-of-australian-citizenship a skos:Concept ;
+ skos:definition "An Australian citizenship certificate is an official document that shows that a person is an Australian."@en ;
+ skos:inScheme idt:conceptScheme ;
+ skos:prefLabel "Certificate of Australian citizenship"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:change-of-name-certificate a skos:Concept ;
+ skos:definition "A document issued by a a governing body registering a change of name."@en ;
+ skos:inScheme idt:conceptScheme ;
+ skos:prefLabel "Change of name certificate"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:drivers-license a skos:Concept ;
+ skos:altLabel "Drivers Licence"@en ;
+ skos:definition "The official permit required for a person to legally drive a motor vehicle in Australia. Issued by the States and Territories of Australia."@en ;
+ skos:inScheme idt:conceptScheme ;
+ skos:prefLabel "Drivers license"@en ;
+ skos:topConceptOf idt:conceptScheme .
+
+idt:marriage-certificate a skos:Concept ;

```

Comment:
Not sure that this is a party identifier

Reply:
valid QGOV document